docs/LwM2M_Client/Nordic/Runtime_certificate.md

```diff
@@ -0,0 +1,96 @@
+# Runtime certificate and private key configuration
+Build a project with a runtime certificate and private key.
+
+## Prerequisites
+* The nRF9160 board with a USB cable.
+- Installed **nrfjprog** from [Nordic Semiconductor page](https://www.nordicsemi.com/Products/Development-tools/nrf-command-line-tools/download)
+* Installed **minicom** (for Linux), **RealTerm**, **PuTTy** (for Windows), or another serial communication program.
+* An active [Coiote IoT DM](https://eu.iot.avsystem.cloud/) user account.
+
+## Build and flash the device
+0. Connect the nRF9160 board to a USB port of your machine.
+0. Go to your local **Anjay-zephyr-client** directory
```

Comment:
Since we do not explain how to clone repository we should add another requirements to `Prerequisites` to prepare cloned repository.

Reply:
The link moves you to the README so we don't need to explicitly point this out

Reply:
Someone can think that only following prerequisites on devzone is enough(It was me :grinning: ) so in my opinion pointing out to follow all `README` steps and remove from prerequisities those which are doubled is a good option